subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/git/tasks/IdentifyGitOriginRepoTask.java

```diff
@@ -32,47 +32,41 @@ public IdentifyGitOriginRepoTask() {
 
     @TaskAction
     public void identifyGitOriginRepo() {
-        if (originRepo != null) {
-            LOG.lifecycle("  Using Git origin repository configured on the task: {}", originRepo);
+        if (repository != null) {
```

Comment:
It would be good to have a short comment explaining when this situation can happen.

Reply:
Cool idea. Will do!